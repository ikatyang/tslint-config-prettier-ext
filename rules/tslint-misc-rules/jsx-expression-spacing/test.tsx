const fails = (
  <div>
      {missingOpeningAndClosingSpace}
      {  extraLeadingSpace }
      { extraTrailingSpace  }
  </div>
);
