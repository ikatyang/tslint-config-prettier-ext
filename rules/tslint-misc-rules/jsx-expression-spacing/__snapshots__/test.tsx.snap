// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should affect error message after formatting 1`] = `
"
<<<<<< before
ERROR: (jsx-expression-spacing) /rules/tslint-misc-rules/jsx-expression-spacing/test.tsx[3, 37]: jsx expression should have one space before closing '}'
ERROR: (jsx-expression-spacing) /rules/tslint-misc-rules/jsx-expression-spacing/test.tsx[3, 7]: jsx expression should have one space after opening '{'
ERROR: (jsx-expression-spacing) /rules/tslint-misc-rules/jsx-expression-spacing/test.tsx[4, 7]: jsx expression should have one space after opening '{'
ERROR: (jsx-expression-spacing) /rules/tslint-misc-rules/jsx-expression-spacing/test.tsx[5, 29]: jsx expression should have one space before closing '}'

======
ERROR: (jsx-expression-spacing) /rules/tslint-misc-rules/jsx-expression-spacing/test.tsx[3, 35]: jsx expression should have one space before closing '}'
ERROR: (jsx-expression-spacing) /rules/tslint-misc-rules/jsx-expression-spacing/test.tsx[3, 5]: jsx expression should have one space after opening '{'
ERROR: (jsx-expression-spacing) /rules/tslint-misc-rules/jsx-expression-spacing/test.tsx[4, 23]: jsx expression should have one space before closing '}'
ERROR: (jsx-expression-spacing) /rules/tslint-misc-rules/jsx-expression-spacing/test.tsx[4, 5]: jsx expression should have one space after opening '{'
ERROR: (jsx-expression-spacing) /rules/tslint-misc-rules/jsx-expression-spacing/test.tsx[5, 24]: jsx expression should have one space before closing '}'
ERROR: (jsx-expression-spacing) /rules/tslint-misc-rules/jsx-expression-spacing/test.tsx[5, 5]: jsx expression should have one space after opening '{'

>>>>>> after
"
`;

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
const fails = (
  <div>
      {missingOpeningAndClosingSpace}
      {  extraLeadingSpace }
      { extraTrailingSpace  }
  </div>
);

======
const fails = (
  <div>
    {missingOpeningAndClosingSpace}
    {extraLeadingSpace}
    {extraTrailingSpace}
  </div>
);

>>>>>> after
"
`;
